engine/src/main/java/org/hibernate/search/engine/metadata/impl/ParseContext.java

```diff
@@ -45,6 +46,24 @@ boolean skipFieldBridges() {
 		return indexManagerType == null;
 	}
 
+	/**
+	 * If the {@code IndexManager} type is not defined, we skip the {@code Analyzer} construction.
+	 */
+	boolean skipAnalyzers() {
+		return indexManagerType == null;
```

Comment:
I was wondering if this can really happen.
Shouldn't we have an exception in case the type is null?

But it seems related to existing code, so it is OK forthis  PR